ui/desktop/app/templates/scopes/scope/projects/sessions/session.hbs

```diff
@@ -8,10 +8,16 @@
 <Rose::Layout::Page as |page|>
 
   <page.header>
-    <h1 class='session-details-header'>
+    <span class='session-details-info'>
```

Comment:
FYI a `<span>` is not allow to be the parent of a `<h1>`, [more info about span](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span)

Can we use the `<page.header>` as container? what other patterns are we following in our UI's to achieve this use case?